### PR TITLE
Beta website part I

### PR DESCRIPTION
**Pull request by deblasis**
This PR adds translations for the following pages: 

-  Installation (note: a lot of content here is similar to old website)
    - Start a New React Project
    - Editor Setup

in new BETA website

### CODE REVIEW

File: beta/src/pages/learn/installation.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ---
-title: Installation
+title: Installazione
 ---
 
 <Intro>
 
-React has been designed from the start for gradual adoption. You can use as little or as much React as you need. Whether you want to get a taste of React, add some interactivity to an HTML page, or start a complex React-powered app, this section will help you get started.
+React è stato progettato sin dal principio per essere adottato gradualmente. A seconda dei tuoi gusti, puoi usare il minimo o il maggior numero di funzionalità offerte da React. Se vuoi avere un assaggio di React, aggiungere un po' di interattività ad una pagina HTML, o sviluppare una applicazione complessa basata basata su React, questa sezione ti aiuterà nei primi passi.
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/installation.md:7]:
```suggestion
React è stato progettato sin dal principio per essere adottato gradualmente. A seconda dei tuoi gusti, puoi usare il minimo o il maggior numero di funzionalità offerte da React. Se vuoi avere un assaggio di React, aggiungere un po' di interattività ad una pagina HTML, o sviluppare una applicazione complessa basata su React, questa sezione ti aiuterà nei primi passi.
```

File: beta/src/pages/learn/installation.md [hunk 2]
```diff
@@ -35,23 +35,23 @@ export default function App() {
 
 </Sandpack>
 
-You can edit it directly or open it in a new tab by pressing the "Fork" button in the upper right corner.
+Puoi modificarla direttamente o aprirla in un nuovo tab premendo il bottone "Fork" nell'angolo in alto a destra.
 
-Most pages in the React documentation contain sandboxes like this. Outside of the React documentation, there are many online sandboxes that support React: for example, [CodeSandbox](https://codesandbox.io/s/new), [StackBlitz](https://stackblitz.com/fork/react), or [CodePen](https://codepen.io/pen?&editors=0010&layout=left&prefill_data_id=3f4569d1-1b11-4bce-bd46-89090eed5ddb).
+La maggioranza delle pagine della documentazione di React contengono sandboxes come questa. Al di fuori della documentazione, esistono alcune sandboxes online che supportano Reactç ad esempio, [CodeSandbox](https://codesandbox.io/s/new), [StackBlitz](https://stackblitz.com/fork/react), o [CodePen](https://codepen.io/pen?&editors=0010&layout=left&prefill_data_id=3f4569d1-1b11-4bce-bd46-89090eed5ddb).
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/installation.md:40]:
```suggestion
La maggioranza delle pagine della documentazione di React contengono sandboxes come questa. Al di fuori della documentazione, esistono alcune sandboxes online che supportano React: ad esempio, [CodeSandbox](https://codesandbox.io/s/new), [StackBlitz](https://stackblitz.com/fork/react), o [CodePen](https://codepen.io/pen?&editors=0010&layout=left&prefill_data_id=3f4569d1-1b11-4bce-bd46-89090eed5ddb).
```

File: beta/src/pages/learn/start-a-new-react-project.md [hunk 3]
```diff
@@ -46,38 +46,38 @@ npm start
...
+### Alternative popolari {/*popular-alternatives*/}
 
 * [Vite](https://vitejs.dev/guide/)
 * [Parcel](https://parceljs.org/)
 
-## Building with a full-featured framework {/*building-with-a-full-featured-framework*/}
+## Iniziare con un framework completo {/*building-with-a-full-featured-framework*/}
 
-If you're looking to **start a production-ready project,** [Next.js](https://nextjs.org/) is a great place to start. Next.js is a popular, lightweight framework for static and server‑rendered applications built with React. It comes pre-packaged with features like routing, styling, and server-side rendering, getting your project up and running quickly. 
+Se intendi **iniziare un progetto pronto al rilascio in produzione,** [Next.js](https://nextjs.org/) è un ottimo punto di partenza. Next.js è un popolare framework leggero per la generazione di contenuto statico generato server-side con React. Include funzioni quali routing, styling, e server-side rendering, che ti permetteranno di lanciare il tuo progetto rapidamente.
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/start-a-new-react-project.md:60]:
```suggestion
Se intendi **iniziare un progetto pronto al rilascio in produzione,** [Next.js](https://nextjs.org/) è un ottimo punto di partenza. Next.js è un popolare framework leggero per la generazione di contenuto statico generato server-side con React. Include funzioni quali routing, styling e server-side rendering, che ti permetteranno di lanciare il tuo progetto rapidamente.
```

File: beta/src/pages/learn/start-a-new-react-project.md [hunk 4]
```diff
@@ -46,38 +46,38 @@ npm start
...
+* Un **test runner** che ti permette di eseguire tests sul tuo codice. Test runners popolari: [Jest](https://jestjs.io/).
 
-If you prefer to set up your own JavaScript toolchain from scratch, [check out this guide](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) that re-creates some of the Create React App functionality. A framework will usually also provide a routing and a data fetching solution. In a larger project, you might also want to manage multiple packages in a single repository with a tool like [Nx](https://nx.dev/react) or [Turborepo](https://turborepo.org/).
+Se preferisci impostare la tua toolchain Javascript partendo da zero, [da uno sguardo a questa guida](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) che mostra come ricreare alcune delle funzionalità offerte da Create React App. Un framework offrirà inoltre anche soluzioni per il routing ed il data fetching. Per progetti più grandi, potresti aver bisogno di dover gestire pacchetti multipli in una repository singola usando tools come [Nx](https://nx.dev/react) o [Turborepo](https://turborepo.org/).
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/start-a-new-react-project.md:82]:
```suggestion
Se preferisci impostare la tua toolchain JavaScript partendo da zero, [da uno sguardo a questa guida](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) che mostra come ricreare alcune delle funzionalità offerte da Create React App. Un framework offrirà inoltre anche soluzioni per il routing ed il data fetching. Per progetti più grandi, potresti aver bisogno di dover gestire pacchetti multipli in una repository singola usando tools come [Nx](https://nx.dev/react) o [Turborepo](https://turborepo.org/).
```

**Review by mattia-sanfilippo — CHANGES_REQUESTED** [review]
Ciao Alessandro, mi sono permesso di correggere un po' di typo 😃

File: beta/src/pages/learn/editor-setup.md [hunk 5]
```diff
@@ -4,57 +4,57 @@ title: Editor Setup
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as your write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+In editor correttamente configurato permette una più facile lettura del codice ed una scrittura più veloce. Può anche aiutare nell'individuazione di bugs durante la battitura! Se è la prima volta che configuri un editor o vuoi ritoccare la tua configurazione attuale, abbiamo alcune raccomandazioni.
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/editor-setup.md:7]:
```suggestion
Un editor correttamente configurato permette una più facile lettura del codice ed una scrittura più veloce. Può anche aiutare nell'individuazione di bugs durante la battitura! Se è la prima volta che configuri un editor o vuoi ritoccare la tua configurazione attuale, abbiamo alcune raccomandazioni.
```

File: beta/src/pages/learn/editor-setup.md [hunk 6]
```diff
@@ -4,57 +4,57 @@ title: Editor Setup
...
+* Quali sono gli editor più comuni
+* Come formattare automaticamente il codice
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## Il tuo editor {/*your-editor*/}
 
-[VS Code](https://code.visualstudio.com/) is one of the most popular editors in use today. It has a large marketplace of extensions and integrates well with popular services like GitHub. Most of the features listed below can be added to VS Code as extensions as well, making it highly configurable!
+[VS Code](https://code.visualstudio.com/) è uno degli editor più popolari al momenti. Ha un grande marketplace di estensioni e si integra molto bene con servizi popolari come GitHub. Molte delle funzioni elencate di seguito possono essere aggiunte a VS Code come estensioni, rendendolo altamente configurabile!
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/editor-setup.md:20]:
```suggestion
[VS Code](https://code.visualstudio.com/) è uno degli editor più popolari al momento. Ha un grande marketplace di estensioni e si integra molto bene con servizi popolari come GitHub. Molte delle funzioni elencate di seguito possono essere aggiunte a VS Code come estensioni, rendendolo altamente configurabile!
```

File: beta/src/pages/learn/editor-setup.md [hunk 7]
```diff
@@ -4,57 +4,57 @@ title: Editor Setup
...
+Altri editor di testo popolari in uso nella comunità React sono:
 
-* [WebStorm](https://www.jetbrains.com/webstorm/) is an integrated development environment designed specifically for JavaScript.
-* [Sublime Text](https://www.sublimetext.com/) has support for JSX and TypeScript, [syntax highlighting](https://stackoverflow.com/a/70960574/458193) and autocomplete built in.
-* [Vim](https://www.vim.org/) is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X.
+* [WebStorm](https://www.jetbrains.com/webstorm/) che è un ambiente integrato di sviluppo progettato specificatamente per JavaScript.
+* [Sublime Text](https://www.sublimetext.com/) offre supporto JSX e TypeScript, [evidenziazione della sintassi](https://stackoverflow.com/a/70960574/458193) e autocompletamento.
+* [Vim](https://www.vim.org/) è un editor di testo altamente configurabile progettato per rendere la modifica di ogni tipo di testo molto efficiente. É incluso nella maggioranza delle distribuzoni UNIX e Apple OS X lanciando il comando "vi".
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/editor-setup.md:26]:
```suggestion
* [Vim](https://www.vim.org/) è un editor di testo altamente configurabile progettato per rendere la modifica di ogni tipo di testo molto efficiente. É incluso nella maggioranza delle distribuzioni UNIX e Apple OS X lanciando il comando "vi".
```

File: beta/src/pages/learn/editor-setup.md [hunk 8]
```diff
@@ -4,57 +4,57 @@ title: Editor Setup
...
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## Funzioni degli editor di testo raccomandate {/*recommended-text-editor-features*/}
 
-Some editors come with these features built in, but others might require adding an extension. Check to see what support your editor of choice provides to be sure!
+Alcuni editor hanno queste funzioni già incluse, altri potrebbero richiedere l'installazione di una estensione. Verifica cosa è supportato dal tuo editor preferito per sicurezza!
 
 ### Linting {/*linting*/}
 
-Code linters find problems in your code as you write, helping you fix them early. [ESLint](https://eslint.org/) is a popular, open source linter for JavaScript. 
+I Code linters individuano problemi nel codice man mano che lo si scrive, permettendo di fixarli sul nascere. [ESLint](https://eslint.org/) è un linter per JavaScript open source molto popolare.
```
Comment (mattia-sanfilippo) [inline beta/src/pages/learn/editor-setup.md:34]:
```suggestion
I code linters individuano problemi nel codice man mano che lo si scrive, permettendo di fixarli sul nascere. [ESLint](https://eslint.org/) è un linter per JavaScript open source molto popolare.
```

**Review by mattia-sanfilippo — CHANGES_REQUESTED** [review]

**Review by LucaBlackDragon — APPROVED** [review]

**Pull request merged**